### DISCUSSION TITLE
c172x doesn't have throttle channel, but it works when throttle command is set to different values. Why?

### DISCUSSION THREAD

**Discussion by xiaominy** [JSBSim-Team/jsbsim#796] [Q&A]
Where is the throttle channel for c172x? The throttle channel is not found in the aircraft/c172x.xml file. Does it mean that the throttle control command does not work on c172x?

However, I set the throttle command to different values, the plane fly different trajectories. Why?

**Comment by seanmcleod** [comment]
Does seem a bit inconsistent, but there appear to be a bunch of pre-defined channels.

https://github.com/JSBSim-Team/jsbsim/blob/36dda4f57cb6b39301f035aec60e17aa83dc61b4/src/models/FGFCS.h#L557-L564

**Reply by bcoconni** [reply]
AFAICS JSBSim copies the throttle position to the throttle command by default so a throttle channel is not needed.
https://github.com/JSBSim-Team/jsbsim/blob/36dda4f57cb6b39301f035aec60e17aa83dc61b4/src/models/FGFCS.cpp#L161-L166
